### ISSUE TITLE
Bug when deleting default game profile

### ISSUE THREAD

**Issue by janezk7** [petermg/TheOcularMigraineMCP#4]
Expected behavior:
- Deleting a game profile delete the profile and **also** disable profile settings if that profile was also set as default game profile

Actual behavior:
- ✅ Deleting a game profile deletes the profile
- ❗ [Bug 1]: If the profile was set as default before deleting, the settings **STILL** activate  when launching the game 
    - ‼️ [Bug 2]: Additionally: when this happens, these settings silently overwrite current settings set in OcularMigraine app and affect other game launches 
   
Bug reproduction:

1.  Set a low resolution in OcularMigraine and set that as a default game profile for a game
2.  Delete that game profile 
3.  Set a high resolution in OcularMigraine and launch the game. The game is expected to now run in this high resolution since its default profile was deleted
4. ❗The game launches with the low resolution, that was previously set in its profile (now deleted)
5. ❗When opening OcularMigraine again and refreshing current set resolution, we see that our high resolution was *OVERWRITTEN* with the old low resolution from the game profile

How to fix game settings after the bug happens:
- Set a new game profile and set as default
- Unset default profile with `Delete default` button  (`Manage game profiles -> [Your default game profile] -> Delete default`)

**Comment by petermg** [comment]
If this is because of what you reported with Ocean Rift, this isn't The Ocular Migraine, it's the actual app itself scaling up the resolution.

**Comment by janezk7** [comment]
I now understand what was the issue with the higher resolution in Ocean Rift, but this issue refers to game resolution not reverting to it's normal behaviour after deleting a default game profile set within The Ocular Migraine app. Please see reproduction steps. It definitely looks like an issue with the app itself.

**Comment by petermg** [comment]
If you are just clicking Delete to delete the profile it will NOT remove the default profile. You have to click on DELETE DEFAULT then the default profile will be deleted.

**Comment by janezk7** [comment]
That's correct. I would suggest that when deleting profile, the app also removes it as default profile if it was set. Otherwise there's no way to revert to original settings unless setting another profile as default and DELETE DEFAULT that one.

**Comment by petermg** [comment]
Clicking on "Default Display Settings" on the Main screen sets it back to original system display settings. If a default profile is set, the "Delete Default" button appears when selecting a saved profile. Having multiple profiles is a feature that allows users to configure profiles for different scenarios and make them default depending on their intended use case at different times as they wish, i.e. one for recording or streaming, another for fastest FPS, another for sharpest image, etc.. Causing the "delete default" button to also delete a users customized profile is contrary to the purpose of having those profiles.

**Comment by janezk7** [comment]
I'm pretty sure there was a misunderstanding. 

petermg:_"Causing the "delete default" button to also delete a users customized profile is contrary to the purpose of having those profiles."_

I suggested the opposite of that. If you "Delete" a customized user's game profile, then that profile should also deactivate (what "Delete Default" button does) IF it was set by default. 

I hope it's clear what I mean. I'm quite certain current behaviour was not intended.

**Comment by petermg** [comment]
Ah. Still, I allow the user to curate their customized profiles, which
includes deleting some if they wish, without forcing them to lose their
default profile. This is why there is a separate delete button. I don't
want to take this option away from users. I will leave it up to the user to
decide which profiles they want to delete.
